repl/src/test/scala/com/cloudera/livy/repl/PythonInterpreterSpec.scala

```diff
@@ -215,3 +221,22 @@ class PythonInterpreterSpec extends BaseInterpreterSpec {
   }
   // scalastyle:on non.ascii.character.disallowed
 }
+
+class Python3InterpreterSpec extends PythonBaseInterpreterSpec {
+
+  implicit val formats = DefaultFormats
+
+  override protected def withFixture(test: NoArgTest): Outcome = {
+    assume(!sys.props.getOrElse("skipPySpark3Tests", "false").toBoolean, "Skipping PySpark3 tests.")
+    test()
+  }
+
+  override def createInterpreter(): Interpreter = PythonInterpreter(new SparkConf(), PySpark3())
+
+  it should "execute `1 / 2` == 0.5" in withInterpreter { interpreter =>
```

Comment:
I know that. But it's not interesting to test that in python 3, "1/2 = 0.5". It is interesting to test that you're really running python 3, and it's much clearer if instead you explicitly check that, instead of testing for some weird language feature.